[2025-01-07T07:44:37.515+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cashier_file_processing.etl_process_restaurant manual__2025-01-07T07:44:35.092117+00:00 [queued]>
[2025-01-07T07:44:37.526+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cashier_file_processing.etl_process_restaurant manual__2025-01-07T07:44:35.092117+00:00 [queued]>
[2025-01-07T07:44:37.526+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-01-07T07:44:37.535+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): etl_process_restaurant> on 2025-01-07 07:44:35.092117+00:00
[2025-01-07T07:44:37.542+0000] {standard_task_runner.py:57} INFO - Started process 493 to run task
[2025-01-07T07:44:37.546+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'cashier_file_processing', 'etl_process_restaurant', 'manual__2025-01-07T07:44:35.092117+00:00', '--job-id', '289', '--raw', '--subdir', 'DAGS_FOLDER/Cashier_file_processing.py', '--cfg-path', '/tmp/tmpyn1xh8i8']
[2025-01-07T07:44:37.548+0000] {standard_task_runner.py:85} INFO - Job 289: Subtask etl_process_restaurant
[2025-01-07T07:44:37.598+0000] {task_command.py:410} INFO - Running <TaskInstance: cashier_file_processing.etl_process_restaurant manual__2025-01-07T07:44:35.092117+00:00 [running]> on host 8fff7f1050bf
[2025-01-07T07:44:37.674+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='cashier_file_processing' AIRFLOW_CTX_TASK_ID='etl_process_restaurant' AIRFLOW_CTX_EXECUTION_DATE='2025-01-07T07:44:35.092117+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-07T07:44:35.092117+00:00'
[2025-01-07T07:44:38.244+0000] {logging_mixin.py:150} INFO - No data found in the 'restaurantmodels' collection.
[2025-01-07T07:44:38.252+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-01-07T07:44:38.286+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=cashier_file_processing, task_id=etl_process_restaurant, execution_date=20250107T074435, start_date=20250107T074437, end_date=20250107T074438
[2025-01-07T07:44:38.359+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-01-07T07:44:38.426+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
